formatting.css

```diff
@@ -0,0 +1,231 @@
+body, html{
+    margin: 0%;
+    padding: 0%;
+}
+.header-banner{
+    position: relative;
+    text-align: center;
+    background-color: darkgrey;
+    width: 100%;
+}
+.header-banner .title-alligner{
+    position: absolute;
+    top: 50%;
+    left: 25%;
+    transform: translate(-50%, -50%);
+    width: 45%;
+}
+.header-banner .cart-alligner{
+    position: absolute;
+    top: 25%;
+    left: 85%;
+    transform: translate(-50%, -50%);

```

Comment:
You could do to get rid of these transforms and rely on top and left alone. But since this is aligned to the right, I'd steer you to right instead of left.